### PR TITLE
Updated release notes for azure-cosmos 4.5.0

### PR DESCRIPTION
**Pull request by kushagraThapar**
* Updated release notes for azure-cosmos 4.5.0

### CODE REVIEW

File: releases/2020-09/java.md [hunk 1]
```diff
@@ -551,6 +551,15 @@ Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here
 
 - Initial preview release for Azure Tables
 
+### Azure Cosmos ([Changelog](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/azure-cosmos/CHANGELOG.md#450-2020-09-16))
```
Comment (alzimmermsft) [inline releases/2020-09/java.md:554]: You'll want to check if the monthly release blog has gone out for September and if so move this to October.

Comment (alzimmermsft) [reply releases/2020-09/java.md:554]: @jongio has the blog gone out?

Comment (jongio) [reply releases/2020-09/java.md:554]: yes

Comment (kushagraThapar) [reply releases/2020-09/java.md:554]: Cool, will update this PR against October release notes, @jongio should we close this PR ?
I will open new one when we have October release notes in progress.

Comment (jongio) [reply releases/2020-09/java.md:554]: yes, sounds good.

**Comment by kushagraThapar** [comment]
Closing this PR since the announcement is already out, I will open new one when we have October release notes in progress.

**Pull request closed without merging**